documentation/src/main/asciidoc/stories/assembly_implementing_server_tasks.adoc

```diff
@@ -0,0 +1,11 @@
+[id='server_tasks-{context}']
+:context: server
+= Implementing Server Tasks
+Server tasks are server-side scripts defined in Java language.
```

Comment:
I would not use the word scripts.
"Server tasks are custom server-side jobs that can be invoked from clients."